drivers/modem/gsm_ppp.c

```diff
@@ -3,6 +3,7 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+#define DT_DRV_COMPAT gsm_ppp
```

Comment:
Add empty line after comment and before `#define ...` statement.

Reply:
Done